### ISSUE TITLE
Match Crawler not picking up any data

### ISSUE THREAD

**Issue by amirorezaee** [hugomathien/football-data-collection#2]
Hey Hugo,

Been trying to run the match crawler to pick up last season's data, however it doesn't seem to be working anymore.

Spitting out empty csv/xml/json files.

Any ideas on why?

**Comment by quelibrio** [comment]
Yes some of the website tables and apis have changed and I worked to resolve this as well as refactoring the code a bit in a fork: https://github.com/quelibrio/football-data-collection here I fixed the game collection, however havn't resolved the players stats collection problems. If someone could cooperate on this tell me to add u as a contributor.

**Comment by adriangasinski** [comment]
Hi. 
@quelibrio  - could you please help to investigate why I can't use your code?

I try to run match crawler but I get this error:
`2017-12-04 14:02:02 [scrapy.core.scraper] ERROR: Spider error processing <GET http://football-data.mx-api.enetscores.com/page/xhr/standings/1/8/0/0/0/0/> (referer: http://football-data.mx-api.enetscores.com/page/xhr/standings/)`

**Comment by quelibrio** [comment]
> football-data



> Hi.
> @quelibrio - could you please help to investigate why I can't use your code?
> 
> I try to run match crawler but I get this error:
> `2017-12-04 14:02:02 [scrapy.core.scraper] ERROR: Spider error processing <GET http://football-data.mx-api.enetscores.com/page/xhr/standings/1/8/0/0/0/0/> (referer: http://football-data.mx-api.enetscores.com/page/xhr/standings/)`

Sory, just seeing this now, change 'football-data' to 'reuters'  The first client stopped paying and is not supported by entscores
